internal/toolexec/aspect/oncompile.go

```diff
@@ -79,7 +84,7 @@ func (w Weaver) OnCompile(cmd *proxy.CompileCommand) error {
 
 		case weaveTracerInternal:
 			log.Debugf("Enabling tracer-internal mode for %q\n", w.ImportPath)
-			shortList := make([]aspect.Aspect, 0, len(aspects))
+			shortList := make([]*aspect.Aspect, 0, len(aspects))
 			for _, aspect := range aspects {
 				if aspect.TracerInternal {
 					shortList = append(shortList, aspect)
```

Comment:
We should be able to simply use `slices.DeleteFunc` here now no ?

Reply:
I guess, but maybe later?